db/migrations/00002_add_confirmation_reset_timeout.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE auth.accounts
+  ADD COLUMN last_sent_confirmation timestamp with time zone DEFAULT now() NOT NULL;
```

Comment:
we use `<verb>_at` ex `updated_at` or `created_at` - so I'm suggesting we use `confirmation_email_sent_at` or `last_confirmation_email_sent_at`.